dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -570,11 +575,21 @@ private void deleteBucket(final Session session, final String bucketName, final
                 protected Object call() throws Exception {
                     try {
                         getClient().deleteBucketSpectraS3(new DeleteBucketSpectraS3Request(bucketName).withForce(true));
+                        Platform.runLater(() -> {
+                            // deepStorageBrowserPresenter.logText("Delete response code: " + deleteBucketSpectraS3Response.getStatusCode(), LogType.SUCCESS);
+                            deepStorageBrowserPresenter.logText("Successfully deleted bucket", LogType.SUCCESS);  });
                     } catch (final IOException e) {
-                        LOG.error("Failed to delte Bucket " + e);
-                        Platform.runLater(() -> deepStorageBrowserPresenter.logText("Failed to delete bucket", LogType.ERROR));
-                        ALERT.setContentText("Failed to delete Bucket");
-                        ALERT.showAndWait();
+                        if (e instanceof FailedRequestException) {
```

Comment:
Instead of using instanceof, use another catch block before the IOException catch block, that has the FaileRequestException logic.